scripts/zephyr_module.py

```diff
@@ -280,7 +285,7 @@ def main():
     while start_modules:
         node = start_modules.pop(0)
         sorted_modules.append(node)
-        node_name = PurePath(node.project).name
+        node_name = node.meta.get('name')
```

Comment:
`node.meta['name']` here too, I think, no?

Reply:
Thanks. Not sure how my search missed that :(
Never do coding in the evening, I guess.